src/ng-backstretch.js

```diff
@@ -21,9 +21,9 @@ directive('backstretch', ['$window', '$timeout', function($window, $timeout) {
        * e.g. <div backstretch backstretch-url="'/path/to/image.jpg'">
        * So, we need to turn this back into an array.
        */
-      scope.images = Array.isArray(scope.images()) ? scope.images() : [scope.images()];
-      scope.duration = scope.duration() || 5000;
-      scope.fade = scope.fade() || 1;
+      scope.images = Array.isArray(scope.images) ? scope.images : [scope.images];
+      scope.duration = !angular.isUndefined(scope.duration) ? scope.duration : 5000;
```

Comment:
I see what's happening. JS evaluates both `0` and `undefined` to be the same which causes the default to be triggered here when either the value is set to `0` of not provided at all. Certainly not expected behavior.

That being said, I just looked at the code for the original jquery backstretch plugin and the default fade should actually be set to `0`. So, in keeping this code as functionally similar to the source, that seems to make the most sense. It would also seem to resolve the issue you're having.

https://github.com/srobbin/jquery-backstretch/blob/master/src/jquery.backstretch.js#L73

So, just changing `scope.fade = scope.fade() || 1;` to `scope.fade = scope.fade() || 0;` should do the trick.

Unfortunately, I'm not quite sure where those pics are from. They're the same images that @srobbin used on the jQuery Backstretch site.